.travis.yml

```diff
@@ -2,6 +2,11 @@ env:
   global:
     - LC_CTYPE=en_US.UTF-8
 
+branches:
```

Comment:
#? why only develop and master, dont we want the branches to build to know whether or not to merge them in?